docs/conf.py

```diff
@@ -113,6 +113,7 @@
     "**/.nox/**/*",
     "samples/AUTHORING_GUIDE.md",
     "samples/CONTRIBUTING.md",
+    "samples/snippets/README.md",
```

Comment:
Rather than having a duplicate `README`, it would be better just to add the language to `samples/snippets/README.rst` (reformatting the ReST as needed).